packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -501,16 +503,35 @@ function APIProvider({ children }) {
       // `?_method=DELETE` is an alternative solution to override the request method.
       // See https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_method-or-x-http-method-override-header
       return apiFetch({
-        path: addQueryArgs(`${customPageTemplates}${id}/`, {
-          _method: 'DELETE',
-        }),
-        data: { force: true },
+        path: addQueryArgs(`${customPageTemplates}${id}/`, {}),
+        data: {},
         method: 'POST',
       });
     },
     [customPageTemplates]
   );
 
+  const getTaxonomies = useCallback(() => {
+    return apiFetch({
+      path: addQueryArgs('/wp/v2/taxonomies/', { type: 'web-story' }),
```

Comment:
All other endpoints, using the web-stories/v1/ namespace, which gives us more control. We should do this here as well @swissspidy ?

Reply:
Perhaps, let's discuss in a separate ticket+PR

Reply:
yea open to whatever. I'm not sure what the difference is between v2 & v2 here

Reply:
We should preload this request on the page.

Reply:
https://github.com/google/web-stories-wp/blob/f62effbfc605a3e2a4ddbc8cfd9fa707db28c57f/includes/templates/admin/edit-story.php#L39-L111

Reply:
would I need to update anything on the client? or does `apiFetch` do all the work to grab a preloaded request if it exists?

Reply:
No, you don't. Yes, it does.

But preloading is a different topic altogether.

@spacedmonkey Could you create separate tickets for these? Thanks

Reply:
I wrote a PR as it was so quick. https://github.com/google/web-stories-wp/pull/9032